storybookjs/storybook
# CLI: Add command to generate annotations file

Issue: N/A

## What I did

The CLI now has a command to generate preset annotations file. This will get framework presets, addon presets and project annotation (preview.js) and write to a file that might look like this:

```
import * as frameworkPreset from "@storybook/react/dist/esm/client/docs/config";
import * as frameworkPreset2 from "@storybook/react/dist/esm/client/preview/config";
import * as addonPreset from "@storybook/addon-docs/dist/esm/frameworks/common/config";
import * as addonPreset2 from "@storybook/addon-actions/dist/esm/preset/addDecorator.js";
import * as addonPreset3 from "@storybook/addon-actions/dist/esm/preset/addArgs.js";
import * as addonPreset4 from "@storybook/addon-backgrounds/dist/esm/preset/addDecorator.js";
import * as addonPreset5 from "@storybook/addon-backgrounds/dist/esm/preset/addParameter.js";
import * as projectAnnotations from "./preview.ts";

export default [addonPreset, frameworkPreset, frameworkPreset2, addonPreset2, addonPreset3, addonPreset4, addonPreset5, projectAnnotations];
```

## How to test

1 - Build CLI
2 - Use it locally with `sb generate-annotations` in a Storybook project


If your answer is yes to any of these, please make sure to include it in your PR.
